openedxscorm/storage.py

```diff
@@ -0,0 +1,37 @@
+"""
+Storage backend for scorm metadata export.
+"""
+from django.core.files.storage import get_storage_class
+from storages.backends.s3boto3 import S3Boto3Storage
+
+
+class S3ScormStorage(S3Boto3Storage):
+    """
+    S3 backend for scorm metadata export
+    """
+    def __init__(self, bucket, querystring_auth, querystring_expire):
+        super().__init__(bucket=bucket, querystring_auth=querystring_auth,
+                         querystring_expire=querystring_expire)
+
+
+def scorm_storage(xblock):
```

Comment:
This function is in the openedxscorm.storage module, so we wan rename it to simply "s3".

```
XBLOCK_SETTINGS["ScormXBlock"] = {
    "STORAGE_FUNC": "openedxscorm.storage.s3"
}
```